task/fbc-related-image-check/0.1/fbc-related-image-check.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
   name: fbc-related-image-check
 spec:
   description: >-
-    Checks whether all images referenced in file-based catalog (FBC) are valid.
+    Checks whether all images referenced in file-based catalog (FBC) are valid by using Skopeo to inspect manifest content.
```

Comment:
Suggestion: Check the validity of all the images referenced in the file-based catalog (FBC) to inspect manifest content with Skopeo.

BTW what do we mean by "inspect manifest content"? Depending on this we may have to update the description again.

Reply:
> BTW what do we mean by "inspect manifest content"?

We run the command `skopeo inspect <image-name>` which returns low-level information about the image like the layers that make up the image, the platforms for which the image is intended, and other metadata. If this command returns any failure, we record it and consider the image invalid.